flexmeasures/api/v3_0/__init__.py

```diff
@@ -11,6 +12,21 @@ def register_at(app: Flask):
 
     v3_0_api_prefix = "/api/v3_0"
 
+    def cost_function() -> int:
+        if request.endpoint == "SensorAPI:trigger_schedule":
```

Comment:
I'm afraid that if for some reason the endpoint's identification string changes, we suddenly stop limiting it.

Reply:
I can probably find a way to obtain this string in a way that directly references the class method.

`SensorAPI.__name__ + ":" + SensorAPI.trigger_schedule.__name__` would be my first try.